demos/projects/ESPRESSIF/aziotkit/main/azure_iot_freertos_esp32_sensors_data.h

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
```

Comment:
nit: C banner in all file part of this PR

Reply:
Fixed.

Reply:
Here and in other files of the sample.